bin/pinga/src/args.rs

```diff
@@ -119,6 +119,10 @@ pub(crate) struct Args {
     #[arg(long)]
     pub(crate) concurrency: Option<u32>,
 
+    /// Enables NATS JetStream consumer(s) to have poison pill protection
+    #[arg(long)]
+    pub(crate) jetstream_consumer_poison_pill_protection: Option<bool>,
```

Comment:
This is an "opt-out" option. If it gives us trouble, we can disable it without a git-based deployment.